R/qvcalc.itempar.R

```diff
@@ -1,5 +1,6 @@
 qvcalc.itempar <- function(object, ...){
-  if (!(attr(object, "alias"))) stop(
+  alias = attr(object, "alias")
```

Comment:
I found that if `alias = FALSE` in `itempar`, then the attribute is not logical, but something else, so used `isTRUE` here.